solutions/swb-reference/README.md

```diff
@@ -6,348 +6,728 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
-2. Install Docker ([link](https://docs.docker.com/get-docker/))
+# Setup Instructions for SWBv2p1
+
+Follow the instructions below to deploy SWBv2 with Sagemaker Notebook lifecycle support. You'll be able to 
+launch/start/stop/terminate/connect to a Sagemaker Notebook instance.
+
+## Prerequisite
+* An AWS account for deploying SWBv2 API. This account will be called the `Main Account`.
+   * On the command line, set your [credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) to have your `Main Account` as the `default` profile 
+* An AWS account for hosting environments. This account will be called the `Hosting Account`.
+* Software
+  * [Rush](https://rushjs.io/pages/developer/new_developer/) v5.62.1 or later. We'll be using this tool to manage the packages in our mono-repo
+  * Node 14.x or 16.x [(compatible node versions)](https://github.com/aws-solutions/solution-spark-on-aws/blob/main/rush.json#L9)
+  * [POSTMAN](https://www.postman.com/) (Optional) This is used for making API requests to the server. POSTMAN is not needed if you already have a preferred API client. 
+* The requirements below are for running the lambda locally 
+   * Install SAM CLI ([link](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html))
+   * Install Docker ([link](https://docs.docker.com/get-docker/))
+
+## Installation
+### Setup Config File
+1. Navigate to `solutions/swb-reference`.
+1. Copy `src/config/example.yaml` and create a new file in the format `<STAGE>.yaml` in the config folder. The stage value uniquely identifies this deployment. Some common values that can be used are `dev`, `beta`, and `gamma`.
+1. Open your new `<STAGE>.yaml` file and uncomment the `stage` attribute. Provide the correct `<STAGE>` value for the attribute
+1. Open your new `<STAGE>.yaml` file and uncomment `awsRegion` and `awsRegionShortName`. `aws-region` value can be one of the values on this [table](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html#Concepts.RegionsAndAvailabilityZones.Regions), under the `Region` column. `awsRegionName` can be a two or three letter abbreviation for that region, of your own choosing. The `awsRegion` value will determine which region SWBv2 is deployed in.
+1. Uncomment `rootUserEmailParamStorePath` and provide a name for a SSM parameter that will contain the main account user's email address, e.g. `/swb/<stage>/rootUser/email`.
+1. Follow instructions to [create a SSM Parameter](https://docs.aws.amazon.com/systems-manager/latest/userguide/parameter-create-console.html) in your main account and set the name as the assigned value in `rootUserEmailParamStorePath` and the value as the main account user's email address.
+1. (Optional) Uncomment `allowedOrigins` and provide a list of URLs that will be allowed to access your SWB API, e.g. ['http://localhost:3000','http://localhost:3002'].
```

Comment:
Fairly certain deployment fails if this is commented out. Have you successfully deployed with it commented out?

Reply:
just tested, it cannot be taken out or left empty, so definitely not optional

Reply:
Updateed to non optional